File: app/src/main/java/org/deiverbum/app/model/BibleBook.java
package org.deiverbum.app.model;

import static org.deiverbum.app.utils.Utils.normalizeEnd;

/**
 * <p>Nueva clase para manejar libros de la Biblia en el contexto litúrgico.</p>
 * @author A. Cedano
 * @version 1.0
 * @since 2022.2
 */
public class BibleBook {
    private Integer id;
    private String name;
    private String liturgyName;
    private String shortName;

    public BibleBook() {
    }


    public String getName() {

        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getShortName() {
        return shortName;
    }

    public void setShortName(String shortName) {
        this.shortName = shortName;
    }

    public String getLiturgyName() {
        return liturgyName;
    }


    public void setLiturgyName(String liturgyName) {
        this.liturgyName = liturgyName;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }


    public String getForRead() {
        return normalizeEnd(getLiturgyName());
    }
}

File: app/src/main/java/org/deiverbum/app/model/BibleBook.kt
package org.deiverbum.app.model

import org.deiverbum.app.util.Utils

/**
 *
 * Nueva clase para manejar libros de la Biblia en el contexto litúrgico.
 *
 * @author A. Cedano
 * @version 1.0
 * @since 2023.1
 */
class BibleBook {
    var id: Int? = null
    var name: String? = null
    var liturgyName: String? = null
    var shortName: String? = null

    fun getForRead(): String {
        return Utils.normalizeEnd(liturgyName)
    }
}